common/src/main/java/org/opensearch/ml/common/transport/model/MLUpdateModelInput.java

```diff
@@ -97,6 +101,9 @@ public MLUpdateModelInput(StreamInput in) throws IOException {
             connector = new MLCreateConnectorInput(in);
         }
         lastUpdateTime = in.readOptionalInstant();
+        if (in.readBoolean()) {
```

Comment:
@jngz-es This is one of the *Input file, you may need to add the version filter here?

Reply:
NP, just curious, why rate limiter no version filter here?

Reply:
discussed with @b4sjoo , I see now, the update model api is new in 2.12.

Reply:
added